apps/web/src/app/components/CollateralTypeSwitcher/MultiChainItem.tsx

```diff
@@ -0,0 +1,68 @@
+import { Token } from '@balancednetwork/sdk-core';
+import { Position, XChainId } from 'app/pages/trade/bridge/types';
+import React from 'react';
+import CurrencyLogo from '../CurrencyLogo';
+import { Typography } from 'app/theme';
+import SingleChainItem from './SingleChainItem';
+import { AssetSymbol, BalanceAndValueWrap, BalanceBreakdown, ListItem } from '../Wallet/styledComponents';
+import styled from 'styled-components';
+
+type MultiChainItemProps = {
+  baseToken: Token;
+  positions: Partial<{ [key in XChainId]: Position }>;
+  onSelect: (symbol, chainId?) => void;
```

Comment:
add types

Reply:
moved to https://github.com/balancednetwork/balanced-network-interface/issues/1388